x-pack/legacy/plugins/lens/public/xy_visualization/xy_expression.tsx

```diff
@@ -28,6 +29,10 @@ import {
 import { EuiIcon, EuiText, IconType, EuiSpacer } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
+import { KibanaDatatableColumn } from '../../../../../../src/plugins/expressions/public';
+// this will be imported from vis_type_vislib after merging https://github.com/elastic/kibana/pull/61575
+// @ts-ignore
+import { buildPointSeriesData } from '../../../../../../src/legacy/ui/public/agg_response/point_series/point_series.js';
```

Comment:
We should not import from the vis_type_vislib plugin in lens (and neither ui of course). If this format is very special and we need to always work like that, we should talk to app arch and see if it makes more sense having this function inside the data or expression or uiactions plugins.

Reply:
In the end I decided not to use the function as all, as it adds a lot of complexity and creates structure that is too big and most of information we don't need. I created the structure we use for brushing trigger. It's the same approach as the one we pursue in discover here: https://github.com/elastic/kibana/pull/61575/files